server/chat-plugins/helptickets.ts

```diff
@@ -363,14 +363,16 @@ export class HelpTicket extends Rooms.RoomGame {
 		// skip if the user is autoconfirmed and on a shared ip
 		if (Punishments.sharedIps.has(user.latestIp) && user.autoconfirmed) return false;
 
-		for (const ip of user.ips) {
-			const curPunishment = Punishments.roomIps.get('staff')?.get(ip);
-			if (curPunishment && curPunishment[0] === 'TICKETBAN') {
-				const [, userid,, reason] = curPunishment;
-				return (
-					`You are banned from creating help tickets` +
-					`${userid !== user.id ? `, because you have the same IP as ${userid}` : ''}. ${reason ? `Reason: ${reason}` : ''}`
-				);
+		if (!Config.noipchecks) {
+			for (const ip of user.ips) {
+				const curPunishment = Punishments.roomIps.get('staff')?.get(ip);
+				if (curPunishment && curPunishment[0] === 'TICKETBAN') {
+					const [, userid,, reason] = curPunishment;
+					return (
+						`You are banned from creating help tickets` +
+						`${userid !== user.id ? `, because you have the same IP as ${userid}` : ''}. ${reason ? `Reason: ${reason}` : ''}`
+					);
+				}
```

Comment:
I thought we agreed to still apply ip checks to punishments?

Reply:
Oh, forgot to remove this one. One sec.